<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-A9SHFLG8" name="RegisterTest" tests="1" failures="1" timestamp="16 Sep 2017 18:31:13 GMT" time="0.046" errors="0">
  <testcase name="registerTest" time="0.004" classname="Testcases.RegTest">
    <failure type="java.lang.AssertionError" message="error message expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: error message expected [true] but found [false]
	at Testcases.RegTest.registerTest(RegTest.java:16)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- registerTest -->
</testsuite> <!-- RegisterTest -->
